--- pimatic.js.orig	2016-02-27 20:06:51.700515097 +0000
+++ pimatic.js	2016-02-27 20:23:03.368922995 +0000
@@ -11,12 +11,18 @@
   process.exit(1);
 }
 
+initialize = function () {
+  require('./startup').startup();
+};
+
 run = function () {
   require('./startup').startup().done();
 };
 
 var command = process.argv[2];
-if(!command || command === "run") {
+if(command === "initialize") {
+  initlialize();
+} else if(!command || command === "run") {
   run();
 } else {
   logFile = path.resolve(__dirname, '../../pimatic-daemon.log');
